### ISSUE TITLE
Stop download/upload only after all threads completely uploaded/downloaded after CTRL+C (next best termination instead of instant termination)

### ISSUE THREAD

**Issue by rtxbae** [rclone/rclone#6218]
#### The associated forum post URL from `https://forum.rclone.org`
None

#### What is your current rclone version (output from `rclone version`)?
v1.58.1

#### What problem are you are trying to solve?
During transferring, when pressing CTRL+C, rclone will terminates all halfway transferred operations.

#### How do you think rclone should be changed to solve that?
Instead of killing all of the operations instantly (threads), wait until all the currently running process reached 100%, then gracefully terminate. Perhaps using another exit key like CTRL+Z.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by ncw** [comment]
I think this is a duplicate of https://github.com/rclone/rclone/issues/966 - can you subscribe to that issue please - thank you.

**Comment by rtxbae** [comment]
> I think this is a duplicate of https://github.com/rclone/rclone/issues/966 - can you subscribe to that issue please - thank you.

Wow didn't know the idea came up 6 years ago.
